### ISSUE TITLE
About the BatchNorm layer setting

### ISSUE THREAD

**Issue by Parsifal133** [ihaeyong/WSN#3]
Hi, @ihaeyong! I noticed that you set track_running_stats to False, in nn.BatchNorm2d layer, which is the same as The GPM does (ICLR 2021, pretty sure that you have read it ).
Would you please illustrate this setting briefly?
![736b856a7b5320fa120b232b87b52ee](https://user-images.githubusercontent.com/55016754/230567694-ea100a4f-34c4-4994-8de1-3957bbaa04ea.png)

**Comment by Parsifal133** [comment]
I noticed that setting the track_ running_ state to True will record the mean and variance of the new task data, thereby hindering the inference of the old task.
